### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by NorfairKing** [jfischoff/tmp-postgres#281]
Stackage is blocked on a release of tmp-postgres: https://github.com/commercialhaskell/stackage/issues/7140

**Comment by NorfairKing** [comment]
This is blocking stackage.
It would be great to get a release in to get the stackage packages enabled again.

**Comment by lambdaTotoro** [comment]
+1 on this, would be nice to have this resolved.
Especially since the necessary code seems to already exist.

**Comment by peterbecich** [comment]
This would also help to fix `tmp-postgres` in NixPkgs.

The new release with https://github.com/jfischoff/tmp-postgres/commit/7f2467a6d6d5f6db7eed59919a6773fe006cf22b will probably fix:
```
NIXPKGS_ALLOW_BROKEN=1 nix build nixpkgs#haskellPackages.tmp-postgres --impure 
```

```
.
.
.
       > Error: Setup: Encountered missing or private dependencies:
       > ansi-wl-pprint <1
.
.
.
```

Thanks
